src/app/component/layout/components/search-all-results/search-all-results.component.scss

```diff
@@ -136,3 +136,39 @@ $main-font-color: #494A49;
     }
   }
 }
+
```

Comment:
Why did you use rem? Mix up px rem and % is bad practice.

Reply:
agree

Reply:
fixed